### PR TITLE
Bug 1964380: azure upi: do not create etcd DNS records

### PR DESCRIPTION
**Pull request by MayXuQQ**
Bug 1964380 [DOC] Remove the etcd DNS records

### CODE REVIEW

File: upi/azure/05_masters.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     },
     "privateDNSZoneName" : {
       "type" : "string",
+      "defaultValue" : "privateDNSZone",
```
Comment (jianlinliu) [inline upi/azure/05_masters.json:35]: I think empty string - "" as default value should be better here.

File: upi/azure/05_masters.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     },
     "privateDNSZoneName" : {
       "type" : "string",
+      "defaultValue" : "",
       "metadata" : {
         "description" : "Name of the private DNS zone the master nodes are going to be attached to"
```
Comment (staebler) [inline upi/azure/05_masters.json:37]: Change the description to indicate that this field is not used.
```suggestion
        "description" : "unused"
```

**Comment by staebler** [comment]
The changes look good to me. Unfortunately, the e2e-azure-upi test is broken until https://github.com/openshift/release/pull/22320 lands.

**Comment by MayXuQQ** [comment]
/assign staebler 
> The changes look good to me. Unfortunately, the e2e-azure-upi test is broken until [openshift/release#22320](https://github.com/openshift/release/pull/22320) lands.

My pr need waiting for 22320 ?

**Pull request merged**